src/openforms/registrations/contrib/demo/plugin.py

```diff
@@ -41,3 +44,10 @@ def get_reference_from_result(self, result: None) -> NoReturn:
 
     def update_payment_status(self, submission: "Submission"):
         pass
+
+    def test_config():
+        test = []
+
+        test.append({'completed': False, 'error': 'geen fout melding', 'msg': 'Iets ging fout', 'name': 'name'})
```

Comment:
So, about the return value for `test_config`.

It's probably nicer to return `True` if all works as expected and raise (a self defined) `ConfigException` using the error message:

```
if plugin is not configured:
  return False
if plugin test fails:
   raise ConfigException("failure message")
return True
```